src/main/java/org/benetech/servicenet/service/impl/ActivityServiceImpl.java

```diff
@@ -47,13 +48,14 @@ public ActivityServiceImpl(OrganizationService organizationService, ConflictServ
     @Override
     public List<ActivityDTO> getAllOrganizationActivities(UUID systemAccountId) {
         List<ActivityDTO> activities = new ArrayList<>();
-        List<OrganizationDTO> orgs = organizationService.findAllWithOwnerId(systemAccountId);
+        //List<OrganizationDTO> orgs = organizationService.findAllWithOwnerId(systemAccountId);
+        List<OrganizationDTO> orgs = organizationService.findAll();
 
         // TODO: get organization-entities mapping for every organization
         // this mock below will be removed when Organization matching will be available
         List<UUID> entities = Collections.singletonList(UUID.fromString(EXAMPLE_DATA));
         UUID orgId = null;
-        if (orgs.get(0) != null) {
+        if (!CollectionUtils.isEmpty(orgs) && orgs.get(0) != null) {
```

Comment:
I think orgs.get(0) will never be null

Reply:
It's a part of tmp mock so I'll leave it just in case